tests/integration/backup_tests/test_backups.py

```diff
@@ -54,7 +59,7 @@ async def test_build_and_deploy(ops_test: OpsTest) -> None:
         await ops_test.model.deploy(db_charm, num_units=3)
 
     # deploy the s3 integrator charm
-    await ops_test.model.deploy(S3_APP_NAME, channel="edge", revision=17)
+    await ops_test.model.deploy(S3_APP_NAME, channel="edge", revision=24)
```

Comment:
lets remove revision, and ensure it works on latest in our tests 🙏 

sorry, I know it wasn't you who pinned the revision in the first place, it was me 😬